lib/mnesia/src/mnesia_loader.erl

```diff
@@ -210,7 +210,19 @@ do_get_network_copy(Tab, _Reason, _Ns, unknown, _Cs) ->
     verbose("Local table copy of ~tp has recently been deleted, ignored.~n", [Tab]),
     {not_loaded, storage_unknown};
 do_get_network_copy(Tab, Reason, Ns, Storage, Cs) ->
-    [Node | Tail] = Ns,
+    [Node | Tail] =
```

Comment:
Question: How can we be sure `Ns` is 'randomly' ordered

Reply:
Question: If the 'master' node is present, is it `hd(Ns)` ?

Reply:
> Question: How can we be sure `Ns` is 'randomly' ordered
Yes,  the head of node list is random

> Question: If the 'master' node is present, is it `hd(Ns)` ?
 I should say it could be a list of Master nodes.

The node list we get here is already filtered,  that is why I double check the membership  that if the specified node is in the filtered list, for filtering, check`mnesia:controller:filter_active/1` for more.